### ISSUE TITLE
Support for Proof of Address document uploads

### ISSUE THREAD

**Issue by SKLn-Rad** [onfido/onfido-android-sdk#121]
Sorry for not using the template but this doesnt really fit!
I am hoping this might be just something obvious I am missing, but we are wrapping your Android and iOS SDK for a Flutter application we are developing, and part of the flow is the need to upload proof of address documentation through your SDK. Specifically the ones listed in your v2 Restful API seen here.

[https://documentation.onfido.com/v2/#upload-document](url)

It seems the SDK only has options for uploading non-country specific documents such as the usual passports, driving licenses, etc.

In short:
Can we upload a POA document through the native SDKs, or do we have to rely on uploading through our backend.

**Comment by ramprasad-parajuli** [comment]
Hi @SKLn-Rad Android SDK do not support upload of POA documents and you will have to upload the POA documents through your backend.

**Comment by SKLn-Rad** [comment]
No worries, thanks for the quick reply.
